hummingbot/connector/exchange/bybit/bybit_exchange.py

```diff
@@ -192,36 +210,45 @@ async def _place_order(self,
 
         side_str = CONSTANTS.SIDE_BUY if trade_type is TradeType.BUY else CONSTANTS.SIDE_SELL
         symbol = await self.exchange_symbol_associated_to_pair(trading_pair=trading_pair)
-        api_params = {"symbol": symbol,
+        api_params = {"category": "spot",
+                      "symbol": symbol,
                       "side": side_str,
+                      "orderType": type_str,
                       "qty": amount_str,
-                      "type": type_str,
                       "orderLinkId": order_id}
         if order_type != OrderType.MARKET:
             api_params["price"] = f"{price:f}"
         if order_type == OrderType.LIMIT:
             api_params["timeInForce"] = CONSTANTS.TIME_IN_FORCE_GTC
 
         order_result = await self._api_post(
-            path_url=CONSTANTS.ORDER_PATH_URL,
+            path_url=CONSTANTS.ORDER_PLACE_PATH_URL,
             params=api_params,
             is_auth_required=True,
             trading_pair=trading_pair,
             headers={"referer": CONSTANTS.HBOT_BROKER_ID},
         )
 
-        o_id = str(order_result["result"]["orderId"])
-        transact_time = int(order_result["result"]["transactTime"]) * 1e-3
+        try:
+            o_id = str(order_result["result"]["orderId"])
+            transact_time = int(order_result["result"]["transactTime"]) * 1e-3
+        except Exception as e:
+            tracebakc.print_exc()
```

Comment:
traceback, typo? also I don't see traceback not imported here. is this fine?

Reply:
traceback typo

Reply:
Corrected, and this time lint was much more strict and I fixed a bunch of other things too. Not sure as to why lint did not warn me the first time round.